packages/wrangler/src/__tests__/find-additional-modules.test.ts

```diff
@@ -191,6 +196,59 @@ describe("traverse module graph", () => {
 		expect(modules.length).toStrictEqual(0);
 	});
 
+	it("should ignore Wrangler files", async () => {
+		await mkdir("./src", { recursive: true });
+		await writeFile(
+			"./src/index.js",
+			dedent/* javascript */ `
+			import { HELLO } from "./other.js"
+			export default {
+				async fetch(request) {
+					return new Response(HELLO)
+				}
+			}
+			`
+		);
+		await writeFile(
+			"./src/wrangler.jsonc",
+			dedent/* jsonc */ `
+			{
+				"compatibility_date": "2025/01/01"
+			}
+			`
+		);
+
+		await mkdir("./src/.wrangler/tmp", { recursive: true });
+		await writeFile(
+			"./src/.wrangler/tmp/temp-file.js",
+			dedent`
+			export const DO_NOT_BUNDLE = 10;
+			`
+		);
+
+		const modules = await findAdditionalModules(
+			{
+				file: path.join(process.cwd(), "./src/index.js"),
+				projectRoot: path.join(process.cwd(), "./src"),
+				configPath: path.join(process.cwd(), "./src/wrangler.jsonc"),
+				format: "modules",
+				// The default module root is dirname(file)
+				moduleRoot: path.join(process.cwd(), "./src"),
+				exports: [],
+			},
+			[
+				{ type: "ESModule", globs: ["**/*.js"] },
+				{ type: "Text", globs: ["**/*.jsonc"] },
+			]
+		);
+
+		expect(modules.map((m) => m.name)).toMatchInlineSnapshot(`
+			Array [
+			  "wrangler.jsonc",
```

Comment:
Shouldn't this be ignored?

Reply:
absolutely!

Reply:
36848452fc8bf1cce5352ea3c73fc3c5c3a95ef5